### ISSUE TITLE
how to read the rotation of layer?

### ISSUE THREAD

**Issue by dogeFu** [Agamnentzar/ag-psd#161]
![image](https://github.com/Agamnentzar/ag-psd/assets/23008916/decf6aa9-9e81-4aef-9bbe-0a9baa2e34fd)

I read the readme_psd.md，failed to find where to get to rotaion of a layer,can you help me?

**Comment by Agamnentzar** [comment]
I don't think this value is stored anywhere in PSD file, it's calculated back from the transform (`placedLayer.transform`) that specifies location of 4 corners of transformation cage.

**Comment by zms664011036** [comment]
hello, transform= [12391.430367826988, 1721.516648354885, 2447.364078652975, 8802.809486322956, -70.25686706304622, 5075.064425643115, 9873.809422110964, -2006.2284123249551];) The four corners, how to calculate the rotation Angle, can provide the code to calculate the Angle?
